### PR TITLE
[Debian Updater]: Added support to create install and update partitions

### PR DESCRIPTION
**Pull request by YuriiHordynskyi**
* Added support to create install and update partitions
* Modified GRUB configuration file to show ISS related data on boot menu

### CODE REVIEW

**Comment by YuriiHordynskyi** [comment]
@vitalivanov @oleksandrivantsiv,
I am not able to add you thru "create pull request" procedure.
Could you please review and comment this PR?

**Comment by vitalivanov** [comment]
previously that script was universal for our sdk and iss images...
now these changes are making it to be iss specific only...

**Comment by vitalivanov** [comment]
i tried to identify installer type based on presence of iss binary like here:
https://github.com/InterfaceMasters/onie-x86-installer/commit/c296a6710304d0609df8e66fa4233b7b713922a7

File: components/os-installer.sh [hunk 1]
```diff
@@ -151,6 +178,8 @@ cp -a /mnt/lib64 /lib64
 cp -a /mnt/sbin/e2fsck /tmp/utils/
 cp -a /mnt/sbin/fsck.ext2 /tmp/utils/
 cp -a /mnt/sbin/fsck.ext4 /tmp/utils/
+cp -a ./curl /mnt/sbin/
+cp -a ./partclone.restore /mnt/sbin/
```
Comment (vitalivanov) [inline components/os-installer.sh]: hmm, that may be dangerous

Comment (vitalivanov) [reply components/os-installer.sh]: why do you do that?

Comment (YuriiHordynskyi) [reply components/os-installer.sh]: Will remove. I used this part of code while debugging and testing and forgot to remove.

**Comment by YuriiHordynskyi** [comment]
> i tried to identify installer type based on presence of iss binary like here:
> [c296a67](https://github.com/InterfaceMasters/onie-x86-installer/commit/c296a6710304d0609df8e66fa4233b7b713922a7)

**Comment by YuriiHordynskyi** [comment]
> previously that script was universal for our sdk and iss images...
> now these changes are making it to be iss specific only...

Thanks. Good point. Will handle that

**Comment by YuriiHordynskyi** [comment]
Related PRs:
https://github.com/IMT-internal/onie/pull/3/files
https://github.com/InterfaceMasters/onie-x86-installer/pull/2/files
https://github.com/IMT-internal/iss/pull/933/files
https://github.com/IMT-internal/bcmbuildctl/pull/107/files

**Review by vitalivanov — APPROVED** [review]
changes are ok

**Pull request merged**